bot/exts/filtering/_settings_types/validations/bypass_roles.py

```diff
@@ -12,7 +12,7 @@ class RoleBypass(ValidationEntry):
     name: ClassVar[str] = "bypass_roles"
     description: ClassVar[str] = "A list of role IDs or role names. Users with these roles will not trigger the filter."
 
-    bypass_roles: set[Union[int, str]]
+    bypass_roles: set[int | str]
```

Comment:
Same as above

Reply:
Reverted in https://github.com/python-discord/bot/pull/2523/commits/65ec4bfe0404d6081e377c274cd34274e7eeb0db